lang/java/reef-tang/tang/src/test/java/org/apache/reef/tang/implementation/avro/TestClassHierarchyRoundTrip.java

```diff
@@ -30,372 +30,520 @@
 import java.io.IOException;
 
 public class TestClassHierarchyRoundTrip extends TestClassHierarchy {
```

Comment:
@jwang98052 Thanks for the suggestion! That makes sense to me. I restored the old RoundTripTest for testing protocol buffer version.